src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute used to binds a parameter to an DocumentDB type.
+    /// </summary>
+    /// <remarks>
+    /// The method parameter type can be one of the following:
+    /// <list type="bullet">
+    /// <item><description><see cref="ICollector{T}"/></description></item>
+    /// <item><description><see cref="IAsyncCollector{T}"/></description></item>
```

Comment:
They could -- I have to supply the proper Converter to convert from the type to string, etc. It's not high-pri for this one as it accepts an object directly so that's the easiest thing to use. But I will file an issue to come back to it.

---
In reply to: [56357361](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56357361) [](ancestors = 56357361)